nestkernel/simulation_manager.h

```diff
@@ -169,7 +169,8 @@ class SimulationManager : public ManagerInterface
   delay to_step_;         //!< update clock_+from_step<=T<clock_+to_step_
   timeval t_slice_begin_; //!< Wall-clock time at the begin of a time slice
   timeval t_slice_end_;   //!< Wall-clock time at the end of time slice
-  long t_real_;     //!< Accumunated wall-clock time spent simulating (in us)
+  long t_real_;   //!< Accumulated wall-clock time spent simulating (in us)
+  bool prepared_; //!< indicates whether the SimulationManager has already been
```

Comment:
`indicates whether the SimulationManager has already been` seems to be missing a word or two.